### PR TITLE
 import: rename Switch to Enter

### PR DESCRIPTION
**Pull request by huachaohuang**
## What have you changed? (mandatory)

Switch is a reserved word in Java, we can't use it.

## What are the type of the changes? (mandatory)

- Bug fix

## How has this PR been tested? (mandatory)

Just rename things.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

File: src/import/engine.rs [hunk 1]
```diff
@@ -102,7 +102,6 @@ fn tune_dboptions_for_bulk_load(opts: &DbConfig) -> (DBOptions, CFOptions) {
     db_opts.enable_statistics(false);
     // Vector memtable doesn't support concurrent write.
     db_opts.allow_concurrent_memtable_write(false);
-    db_opts.set_use_direct_io_for_flush_and_compaction(true);
```
Comment (huachaohuang) [inline src/import/engine.rs]: This fails tests on tmpfs, and since it is not actually important, I just remove it here.

**Comment by ngaut** [comment]
Can you simply rename switch to switch_mode?

**Comment by huachaohuang** [comment]
Maybe, but I have merged kvproto, can I just stick with this change?

**Comment by ngaut** [comment]
I prefer switch_mode, which is more clear. Could you also change kvproto?

**Comment by huachaohuang** [comment]
OK

**Comment by huachaohuang** [comment]
PTAL

**Pull request merged**